apps/portal/app/components/tags/add-tags.tsx

```diff
@@ -91,12 +91,14 @@ export function AddTags({
     setIsPopoverOpen(false)
   }
 
-  const handleSaveClick = (invalidTag: IdentityPresenter) => {
+  const handleSaveClick = (
+    invalidTag: IdentityPresenter & { tagClaimId: string },
+  ) => {
     setSelectedInvalidTag(invalidTag)
     setSaveListModalActive({
       isOpen: true,
       identity: invalidTag,
-      id: invalidTag.vault_id,
+      id: invalidTag.tagClaimId,
```

Comment:
I'm worried we're going to need to rethink this once the data settles.

This is great for now though, just noting we may need to rethink this a bit